### ISSUE TITLE
Non referenced RCLs

### ISSUE THREAD

**Issue by FrMalan** [isc30/blazor-lazy-loading#46]
Hi,

I am working through the wiki docs and thank you for taking the time to write some docs. I realize these are early days, so I would like to ask:

For a wasm client, can one create RCL modules that are not directly referenced by the wasm project (or a modulesHost project), kind of a "disconnected" setup where I could then build a module from a separate solution and deploy the output to some common modules folder?

**Comment by isc30** [comment]
Yes absolutely! You can reference `BlazorLazyLoading.Module` from an isolated project and then later deploy the `_content` folder of the Module to the Wasm `wwwroot` folder.

There are 2 ways of consuming the module from that point:

#### Manual approach
  Inject `IAssemblyLoader` and call `LoadAssemblyByName`. The module will be [automatically located](https://github.com/isc30/blazor-lazy-loading/blob/master/src/AssemblyLoader/Services/AssemblyDataLocator.cs#L27) based on it's name.

#### Automatic approach
  Use `<Lazy ModuleName="WHATEVER" Name="TYPE_FULL_NAME" />` to force a module discovery and render that component.

If you can give me a more in-detail example of what you are trying to achieve I can give you a more specific answer and even adapt the existing system for such scenarios.

Thanks!
